iml-agent/src/rpm.rs

```diff
@@ -1,25 +1,89 @@
 use regex::Regex;
 
+use std::fmt;
 use std::process::Output;
+use std::result::Result as StdResult;
```

Comment:
Maybe just `use std::result::Result;`?

Reply:
Not changing it now, see the comments on `pub enum Result`.

Reply:
Which comments?  I still am not sure about why this is being renamed.

Reply:
@utopiabound that was a leftover from previous version, removed now